code/obj/item/gun/kinetic.dm

```diff
@@ -1482,6 +1817,69 @@ ABSTRACT_TYPE(/obj/item/survival_rifle_barrel)
 		set_current_projectile(new /datum/projectile/special/spreader/buckshot_burst/scrap)
 		..()
 
+/obj/item/gun/kinetic/striker
+	name = "\improper Striker-7"
+	desc = "A terrifying looking drum shotgun, legally declared as a 'destructive device'."
+	icon = 'icons/obj/items/guns/kinetic48x32.dmi'
+	wear_image_icon = 'icons/mob/clothing/back.dmi'
+	icon_state = "striker12"
+	item_state = "striker"
+	flags =  FPRINT | TABLEPASS | CONDUCT
+	c_flags = EQUIPPED_WHILE_HELD | ONBACK
+	force = MELEE_DMG_RIFLE
+	contraband = 7
+	ammo_cats = list(AMMO_SHOTGUN_ALL)
+	max_ammo_capacity = 7
+	max_move_amount = 1
+	reload_cooldown = 8 DECI SECONDS
+	auto_eject = FALSE
+	can_dual_wield = FALSE
+	two_handed = TRUE
+	has_empty_state = FALSE
+	fire_animation = TRUE
+	default_magazine = /obj/item/ammo/bullets/a12/bird/seven
```

Comment:
The birdshot looks great but now it doesn't seem to eject any shell casings?

Reply:
new bullet type. i forgor. done